### ISSUE TITLE
mismatch output result after convert saved model to onnx model

### ISSUE THREAD

**Issue by hoaquocphan** [onnx/tensorflow-onnx#1428]
Dear All,

I would like to report the issue related to output of the onnx model.

I download the saved model from tensorflow model hub, then convert to onnx model. 
but the output of saved model and onnx model are different.

the procedure for checking as below:

1/ clone my environment from git repo 
`git clone https://github.com/hoaquocphan/shared_files.git`
2/ run command to convert saved model onnx model:
`python3 -m tf2onnx.convert --saved-model imagenet_mobilenet_v1_025_128_classification_4/ --output imagenet_mobilenet_v1_025_128_classification_4.onnx --opset 12 --concrete_function 2 --inputs inputs:0[1,128,128,3] --inputs-as-nchw  inputs:0`
3/ run 2 scripts to run the saved model and onnx model:
`python3 run_onnx_model.py --m imagenet_mobilenet_v1_025_128_classification_4.onnx`
`python3 run_saved_model.py --dir imagenet_mobilenet_v1_025_128_classification_4`

4/ compare 2 outputs of saved model and onnx model and see the difference.

so could anyone know the issue, please help me?

**Comment by TomWildenhain-Microsoft** [comment]
Hi @hoaquocphan 

It seems that the preprocessing (and possibly resizing) of the image is causing the input to the model to differ.  I replaced the lines in your **run_saved_model** script:
`detections = detections.numpy()`
with:
```
import onnxruntime as ort
sess = ort.InferenceSession("path/to/model.onnx")
inp = input_tensor.numpy()
inp = np.moveaxis(inp, 3, 1)
res = sess.run(None, {"inputs:0": inp})
detections = res[0]
```
and I get identical results to TF.  To convert the model, I used:
`python3 -m tf2onnx.convert --saved-model imagenet_mobilenet_v1_025_128_classification_4/ --output path/to/model.onnx --opset 12 --concrete_function 2 --inputs inputs:0 --inputs-as-nchw inputs:0`
This is the same command you used but I removed the shape constraint so resizing the input is not necessary.
